src/App.tsx

```diff
@@ -4,7 +4,7 @@ import { createGlobalStyle } from 'styled-components'
 
 import Game from './components/Game'
 import { Game as GameLogic, BasicGameScenario } from './game'
-import { WorldState } from './game/ContentTypes'
+import * as GameWorldLoader from './game/GameWorldLoader'
```

Comment:
Why are some files of the `/game` module not exported through `index` while others are?

Any optimizations to allow tree shaking or similar? Or could these imports be combined so everything from the `game` directory is available to import together?

Reply:
Moving forward I might want to separate loaders and data format specifications from 'game' entirely but I wasn't sure how to do it nicely without creating to many changes at the same time.